### ISSUE TITLE
More examples in read me please

### ISSUE THREAD

**Issue by bubnenkoff** [mleise/fast#3]
Marco, big thanks for your Json parser, but could you add move examples in readme of how it's can be used?

**Comment by mleise** [comment]
Yes documentation is a problem. I've seen people generate documentation from source code comments and upload that to their website. That way you don't need to keep source comments and a readme in sync. Also when the library grows the readme will become monolithic, so I kept it down to some basic examples for now.
Anyways what you can expect is only JSON deserialization either from file or a string. Normally it checks for correctness the parts of JSON you use, but not those you skip (with skipValue or singleKey). You can also configure it in 2 other ways with a compile-time argument: trustedSource = assume correct JSON, don't validate; validateAll = Acts like most JSON parsers and validates everything during parsing. If you only want to validate and not use the JSON data, there are convenience functions for that.
